data/schema/schema_units.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
- I think we should just name this file `units.json` as it's already in a `schema` folder
- I think the first line should be `"$schema": "http://json-schema.org/draft-04/schema"`
- `unit` should have a type of `string`